CHANGELOG.md

```diff
@@ -32,9 +32,12 @@
 * reimplemented prometheus metrics exporter to provide gauges, histograms and counter metrics
 * removed shared counter, because it is redundant to the metrics
 * get exception stack trace by setting environment variable `DEBUG`
+* improve loading times for the rule tree by optimizing the rule segmentation and sorting
 
 ### Bugfix
 
+* Fix the rule tree parsing some rules incorrectly, potentially resulting in more matches.
```

Comment:
version 8.0.0 is already released. please push your impovement up to the next section